førsteukast: finne data (#3)
fellesdatakatalog/info-pages-playground: getting-started-finding-data/finding-data.md

# Finne data

> Felles datakatalog (FDK) gir en omfattende oversikt over datasett, begreper, API-er, informasjonsmodeller, tjenester og hendelser i offentlig sektor, inkludert data med juridiske begrensninger som taushetsplikt eller personopplysninger. For å finne frem blant alle beskrivelsene, tilbyr FDK forskjellige måter å søke på. Dersom du ikke finner dataen du leter etter, kan du etterspørre dem. Det angis om dataene er en autoritativ kilde, og det vises om dataene har "allmenn tilgang" eller om det kreves hjemmel eller samtykke for å ta de i bruk.

Reviewer: ```suggestion
Data.norge.no gir en omfattende oversikt over datasett, begreper, API-er, informasjonsmodeller, tjenester og hendelser i offentlig sektor, inkludert data med juridiske begrensninger som taushetsplikt eller personopplysninger. Hensikten med denne oversikten er å gjøre det lettere for deg å finne, forstå og ta i bruk data fra offentlig sektor. For å finne frem blant alle beskrivelsene, tilbyr Data.norge.no forskjellige måter å søke på. Dersom du ikke finner dataen du leter etter, kan du registrere en etterspørsel.
```

> Felles datakatalog (FDK) gir en omfattende oversikt over datasett, begreper, API-er, informasjonsmodeller, tjenester og hendelser i offentlig sektor, inkludert data med juridiske begrensninger som taushetsplikt eller personopplysninger. For å finne frem blant alle beskrivelsene, tilbyr FDK forskjellige måter å søke på. Dersom du ikke finner dataen du leter etter, kan du etterspørre dem. Det angis om dataene er en autoritativ kilde, og det vises om dataene har "allmenn tilgang" eller om det kreves hjemmel eller samtykke for å ta de i bruk.

Reviewer: Kunne vi beskrevet dette med autoritativ kilde og åpne data et annet sted?

Author: Hmm ja, gjelder det kun for datasett? Isåfall kunne det kanskje passet på infosiden til datasett?

Reviewer: Jeg tror det bare gjelder datasett, ja. Kan dobbeltsjekke dette før du er tilbake fra ferie.

Reviewer: Har sett gjennom katalogene, og klarer kun å finne denne informasjonen på infosiden til datasett. @NilsOveTen vet du om det er andre kataloger som sier noe om åpne data eller autoritativ kilde?

Reviewer 2: Vi bruker disse per nå kun for datasett, men tror det skal være mulig å definere tilgangsrettigheter for api og tjenester, kanskje også om de er autoritative kilder, men det sier jeg mest fordi det føles korrekt å kunne gjøre det 🤷

Men "Åpne data" vil jeg tro alltid kun vil være forbeholdt datasett.

Reviewer 2: Siden vi uansett sier noe om dette for datasett så er det sikkert best å flytte til infosiden til datasett, og eventuelt skrive noe om det på andre ressurser hvis det blir aktuelt

## Søk

> Felles datakatalog tilbyr tre forskjellige måter å søke etter data på: standard tekstsøk med filtrering, KI-søk og SPARQL-spørring.

Reviewer: ```suggestion
Felles datakatalog tilbyr tre forskjellige måter å søke etter data på: standard tekstsøk med filtrering, Kunstig intelligens-søk og SPARQL-spørring.
```

Author: Er det meningen at det er Kunstig intelligens med stor K?

> ### Kunstig intelligens-søk

Reviewer: ```suggestion
### Kunstig intelligens-søk (KI-søk)
```

> KI-søket bruker metadata fra data.norge.no og gjør det enkelt å finne datasett. Ved å bruke naturlig språk kan du søke fritt uten å måtte kjenne til spesifikke datasettnavn, fagtermer eller tekniske formater.

Reviewer: ```suggestion
KI-søket bruker metadata fra data.norge.no og hjelper deg med å finne datasett. Ved å bruke naturlig språk kan du søke fritt uten å måtte kjenne til spesifikke datasettnavn, fagtermer eller tekniske formater.
```

Author: Er det data.norge.no med stor eller liten D som er mest riktig? Ser vi har blandet litt

Reviewer: Ref chat i slack: Vi går for stor D i første omgang, så endrer vi heller om vi endrer mening senere.

Prøv f.eks. "Hvilke områder i Norge er vernet?"

> Mer teknisk informasjon om KI-søket finner du [her](https://github.com/Informasjonsforvaltning/fdk-llm-search-service).

Reviewer: I følge wcag 2.4.4 skal alle lenkers mål og funksjon fremgår tydelig av lenketeksten.
https://www.uutilsynet.no/veiledning/244-formal-med-lenke-i-kontekst/1251

```suggestion
[Du kan lese teknisk informasjon om KI-søket på data.norge.no her.](https://github.com/Informasjonsforvaltning/fdk-llm-search-service).
```

### Standard søk med filter

> På data.norge.no/search-all finner du et tekstsøk med muligheter for filtrering. Her kan du søke etter titler, beskrivelser og nøkkelord i databeskrivelsene. For å finne de dataene du leter etter, kan du bruke ulike filtre, inkludert filtrering etter ressurstype. Du velger ressurstype ved å bruke fanene under søkefeltet.

Reviewer: Denne bolken kunne vært utbrodert mer. F.eks: Hvilke filtre som er tilgjengelig, sorteringsmuligheter. 
Kanskje passer setningen " Det angis om dataene er en autoritativ kilde, og det vises om dataene har "allmenn tilgang" eller om det kreves hjemmel eller samtykke for å ta de i bruk." her. Eller skulle vi hatt egne sider om de ulike ressurstypene/detaljsidene?

Author: Den er grei, skal få skrevet litt mer her! Ja, vi hadde egentlig tenkt det, så for oss noe sånt: <img width="623" alt="image" src="https://github.com/fellesdatakatalog/info-pages-playground/assets/48259307/d724f6ba-d854-41ca-bb5b-4d4f6d247cca">

### SPARQL

> [SPARQL-tjenesten](https://data.norge.no/sparql) er det mest fleksible og kraftigste verktøyet for å søke i katalogen. Dette er et verktøy hvor du kan utføre egne spørringer i SPARQL på RDF-grafene til Felles datakatalog. Her kan du spørre etter datasett, begreper, API-er og informasjonsmodeller. Du kan også gjøre spørringer direkte til endepunktet, det finner du her: https://sparql.fellesdatakatalog.digdir.no/.

Reviewer: ```suggestion
[SPARQL-tjenesten](https://data.norge.no/sparql) er det kraftigste og mest fleksible verktøyet for å søke i katalogen. Dette er et verktøy hvor du kan utføre egne spørringer i SPARQL på RDF-grafene til data.norge.no. Her kan du spørre etter datasett, begreper, API-er og informasjonsmodeller. Du kan også gjøre spørringer direkte til [SPARQL-endepunktet for data.norge.no](https://sparql.fellesdatakatalog.digdir.no/).
```

#### Hva er SPARQL?

> [SPARQL](https://www.w3.org/TR/rdf-sparql-query/) er et spørrespråk som brukes til å hente data lagret i RDF-format (Resource Description Framework), som er en standardmodell for å beskrive data og ressurser på semantisk web. Les mer om RDF her(lenke).

Reviewer: ```suggestion
[SPARQL](https://www.w3.org/TR/rdf-sparql-query/) er et spørrespråk som brukes til å hente data lagret i RDF-format. RDF Resource Description Framework) er en standardmodell for å beskrive data og ressurser på semantisk web. [Du kan lese mer om RDF her](lenke).
```

#### Eksempel på bruk av SPARQL

Kopiert fra developers handbook, vet ikke om det er et eksempel som passer bedre

Få oversikt over hvilke egenskaper som finnes:

```sparql
PREFIX dcat: <http://www.w3.org/ns/dcat#>
SELECT *
WHERE {
	?dataset a dcat:Dataset .
    ?dataset ?prop ?value.
}
```

Deretter plukke og velge egenskaper:

```sparql
PREFIX dcat: <http://www.w3.org/ns/dcat#>
PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>

> PREFIX purl: <http://purl.org/dc/terms/>

Reviewer: ```suggestion
PREFIX dct: <http://purl.org/dc/terms/>
```

SELECT *
WHERE {​​​​​​​​​​​​​​
 ?dataset a dcat:Dataset .

>  ?dataset purl:title ?title.

Reviewer: ```suggestion
 ?dataset dct:title ?title.
```

 ?dataset rdf:type ?type.
# Velg nye egenskaper ved: ?dataset <ønsket egenskap> ?egenskapsverdi
}
```

> For flere eksempler og informasjon om hvordan du skriver SPARQL-spørringer, kan du lese [her](https://www.w3.org/TR/rdf-sparql-query/#basicpatterns).

Reviewer: ```suggestion
[Her finner du flere eksempler og informasjon om hvordan du skriver SPARQL-spørringer](https://www.w3.org/TR/rdf-sparql-query/#basicpatterns).
```

## Finner du ikke det du leter etter?

> Vet du ikke om data du ser etter finnes? Er det data du trenger som ikke er delt enda? Hvis datasettet ikke er beskrevet i felles datakatalog eller på tilbyders nettsider, bør du gå i dialog med de som tilbyr dem.

Reviewer: ```suggestion
Vet du ikke om data du ser etter finnes? Er det data du trenger som ikke er delt ennå? Hvis datasettet ikke er beskrevet på data.norge.no eller på dataeiers nettsider, bør du gå i dialog med de som tilbyr dem og oppfordre dem til å dele. Du kan også bruke Digdirs tjenester, ved å spørre Datajegeren eller etterspørre datasett og API-er i Datalandsbyen.
```

### Datalandsbyen: Etterspørre datasett og API-er

Datalandsbyen er et nettforum dedikert til dialog mellom tilbydere og konsumenter av data. Her kan du etterlyse datasett og API-er du ønsker tilgang til eller har spørsmål om. Du kan også be om utvidelser av eksisterende datasett og API-er, for eksempel mer tilgjengelig data eller endringer i API-funksjonalitet. Besøk brukerforumet [Datalandsbyen](https://datalandsbyen.norge.no/category/6/ettersp%C3%B8r-datasett-og-api-er) for å legge inn din etterspørsel. En veiledning om hvordan du går frem finner du [her](https://datalandsbyen.norge.no/topic/56/ettersp%C3%B8rr-data-og-api).

> FDK tilbyr en oversikt over alle [publiserte etterspørsler i Datalandsbyen](https://data.norge.no/requests).

Reviewer: ```suggestion
[Data.norge.no tilbyr en oversikt over alle publiserte etterspørsler i Datalandsbyen](https://data.norge.no/requests).
```

#### Om datalandsbyen

> Digdir, Datafabrikken og Statens vegvesen har lansert nettforumet [Datalandsbyen](https://datalandsbyen.norge.no) for å fremme åpen og datadreven innovasjon gjennom samarbeid, dialog og kunnskapsdeling. Dette forumet fungerer som en felles møteplass for alle interesserte i å lære mer om dataforvaltning og ønsker å engasjere seg med både datakonsumenter og datatilbydere.

Reviewer: ```suggestion
Hensikten med nettforumet [Datalandsbyen](https://datalandsbyen.norge.no) er å fremme åpen og datadreven innovasjon gjennom samarbeid, dialog og kunnskapsdeling. Dette forumet fungerer som en felles møteplass for alle som er interesserte i å lære mer om dataforvaltning og ønsker å snakke med både datakonsumenter og datatilbydere.
```

I Datalandsbyen kan ulike brukergrupper inspirere hverandre med nyttige tips og spørsmål, diskutere bruken av data og API-er, samt etterlyse spesifikke datasett de ønsker tilgang til. Forumet gir også mulighet til å vise eksempler på hvordan data benyttes i tjenester og applikasjoner, samt å dele relevante arrangementer.

Datalandsbyen er åpen for alle, men for å delta aktivt må man registrere seg. Målet er å fremme verdiskapende bruk av data for samfunnet gjennom økt kunnskap, åpenhet og innovasjon.

### Datajegeren: Finn og få tilgang til offentlige datasett

> Datajegeren er en tjeneste som hjelper deg med å finne og få tilgang til datasett fra offentlig sektor. Selv om mange datasett er tilgjengelige via Felles Datakatalog, er den ikke komplett. Datajegeren trer inn for å hjelpe bedrifter med å finne og få tilgang til spesifikke datasett.

Reviewer: ```suggestion
Datajegeren er en tjeneste som hjelper deg med å finne og få tilgang til datasett fra offentlig sektor. Selv om mange datasett er tilgjengelige via data.norge.no, er ikke katalogen komplett.
```

> Ved å besøke tjenesten får du tilgang til et enkelt [kontaktskjema](https://data.norge.no/forms/nb/data-hunter). Her kan du beskrive hva du leter etter, med noen felt som er obligatoriske og noen for å spesifisere forespørselen din. Når skjemaet sendes, går det direkte til ansatte i Digitaliseringsdirektoratet som undersøker hvilken etat som eier dataene og tar seg av den nødvendige kontakten for å skaffe dem.

Reviewer: ```suggestion
[Du kan be Datajegeren om hjelp via et enkelt kontaktskjema](https://data.norge.no/forms/nb/data-hunter). Her kan du beskrive hva du leter etter. Noen felt er obligatoriske og noen kan du velge å bruke for å spesifisere forespørselen din. Når skjemaet sendes inn, går det direkte til ansatte i Digitaliseringsdirektoratet som undersøker hvilken etat som eier dataene og tar seg av den nødvendige kontakten for å skaffe dem.
```

## Hvem har ansvar for hva på data.norge.no

> [Skrive noe om hvilken type informasjon virksomhetene er ansvarlig for, og hvilken informasjon Digdir ved data.norge.no er ansvarlig for].

Reviewer: ### Datatilbyderne

Virksomhetene som eier dataene som er beskrevet på data.norge.no har ansvar for selve innholdet i beskrivelsene og for at dataene du får tilgang til stemmer overens med beskrivelsene. 

Hvis du finner noe feil med beskrivelsene eller tilgangen til dataene, kan du kontakte virksomheten som eier beskrivelsen direkte. Kontaktinformasjon vil som regel være knyttet til beskrivelsen. Hvis du er logget inn i Datalandsbyen, kan du også legge igjen en kommentar på selve beskrivelsen på data.norge.no. Denne kommentaren vil vises både på data.norge.no og i Datalandsbyen. Slike kommentarer og påfølgende diskusjon kan være nyttig for andre brukere som trenger de samme dataene som deg. 

### Digitaliseringsdirektoratet

Digitaliseringsdirektoratet har ansvar for informasjon om når beskrivelser ble publisert på data.norge.no, metadatakvalitet, Datajegeren, KI-oversikt, informasjonssider og at de tekniske løsningene på data.norge.no og Datalandsbyen fungerer som de skal. Digitaliseringsdirektoratet følger også opp kommentarer i Datalandsbyen og purrer på virksomheter som ikke svarer opp kommentarer knyttet til beskrivelser i katalogen.

Author: Lagt til 👆

> I Felles datakatalog - data.norge.no ligger de fleste datasett som er tilgjengelige fra offentlige virksomheter. I denne katalogen skal du finne alt du trenger for å oppdage, vurdere og ta i bruk data.

Reviewer: ```suggestion

```

> Usikker på denne delen, dette står allerede i "om fdk" 👇

Reviewer: ```suggestion

```

> Innholdet i Felles datakatalog leveres av ulike offentlige og private virksomheter. Digitaliseringsdirektoratet er ansvarlig for drift og utvikling av nettstedet.

Reviewer: ```suggestion

```

> #

Reviewer: ```suggestion

```

Author: Tanken med denne er at den lager en liten strek mellom forrige avsnitt og det som kommer etterpå (og ikke er relatert) - men det er kanskje misbruk av markdown? 😅

![image](https://github.com/user-attachments/assets/c212b27e-52f9-4a39-8568-15442b35349e)

Reviewer 2: tre bindestreker tror jeg er korrekt syntax for en linje

Reviewer 2: eventuelt tre understreker, husker ikke helt

Reviewer 2: Begge deler skal fungere, i tillegg til tre stjerner:

https://www.markdownguide.org/basic-syntax/#horizontal-rules

> Les mer om å [finne og vurdere data](https://www.digdir.no/datadeling/finne-og-vurdere-data/2256#hvor_kan_du_finne_dataene) på Digitaliseringsdirektoratets nettsider.

Reviewer: ```suggestion
[Du kan lese mer om å finne og vurdere data  på Digitaliseringsdirektoratets nettsider.](https://www.digdir.no/datadeling/finne-og-vurdere-data/2256#hvor_kan_du_finne_dataene)
```

# Finne data

> Data.norge.no gir en omfattende oversikt over datasett, begreper, API-er, informasjonsmodeller, tjenester og hendelser i offentlig sektor, inkludert data med juridiske begrensninger som taushetsplikt eller personopplysninger. Hensikten med denne oversikten er å gjøre det lettere for deg å finne, forstå og ta i bruk data fra offentlig sektor. For å finne frem blant alle beskrivelsene, tilbyr Data.norge.no forskjellige måter å søke på. Dersom du ikke finner dataen du leter etter, kan du registrere en etterspørsel.

Reviewer 2: Kanskje endre "registrere en etterspørsel" til en lenke
```suggestion
Data.norge.no gir en omfattende oversikt over datasett, begreper, API-er, informasjonsmodeller, tjenester og hendelser i offentlig sektor, inkludert data med juridiske begrensninger som taushetsplikt eller personopplysninger. Hensikten med denne oversikten er å gjøre det lettere for deg å finne, forstå og ta i bruk data fra offentlig sektor. For å finne frem blant alle beskrivelsene, tilbyr Data.norge.no forskjellige måter å søke på. Dersom du ikke finner dataen du leter etter, kan du [registrere en etterspørsel](https://data.norge.no/requests).
```

Prøv f.eks. "Hvilke områder i Norge er vernet?"

[Du kan lese teknisk informasjon om KI-søket på data.norge.no her](https://github.com/Informasjonsforvaltning/fdk-llm-search-service).

### Standard søk med filter

På [data.norge.no/search-all](data.norge.no/search-all) finner du et tekstsøk med muligheter for filtrering. Her kan du søke etter titler, beskrivelser og søkeord i databeskrivelsene. For å avgrense søket ditt, kan du bruke ulike filtre. Hvilke filtre som tilbys avhenger av ressurstype. Du velger ressurstype ved å bruke fanene under søkefeltet. Resultatet kan sorteres etter relevans eller publiseringsdato.

### SPARQL

[SPARQL-tjenesten](https://data.norge.no/sparql) er det kraftigste og mest fleksible verktøyet for å søke i katalogen. Dette er et verktøy hvor du kan utføre egne spørringer i SPARQL på RDF-grafene til data.norge.no. Her kan du spørre etter datasett, begreper, API-er og informasjonsmodeller. Du kan også gjøre spørringer direkte til [SPARQL-endepunktet for data.norge.no](https://sparql.fellesdatakatalog.digdir.no/).

#### Hva er SPARQL?

> [SPARQL](https://www.w3.org/TR/rdf-sparql-query/) er et spørrespråk som brukes til å hente data lagret i RDF-format. RDF Resource Description Framework) er en standardmodell for å beskrive data og ressurser på semantisk web. [Du kan lese mer om RDF her](lenke).

Reviewer 2: Usikker på hva som blir best å peke til som ressurs for å lese mer om RDF, men kan vel starte med å peke til hovedsida for RDF hos w3.org og eventuelt endre til noe annet senere
```suggestion
[SPARQL](https://www.w3.org/TR/rdf-sparql-query/) er et spørrespråk som brukes til å hente data lagret i RDF-format. RDF Resource Description Framework) er en standardmodell for å beskrive data og ressurser på semantisk web. [Du kan lese mer om RDF her](https://www.w3.org/RDF).
```

Reviewer: Enten som Nils Ove beskriver, eller om vi skal ha en egen side om RDF for å forklare det de trenger å vite på en enkel måte. F.eks. ta utgangspunkt i @terjesyl sin beskrivelse i ressursheftet til SIKT-konferansen.

Author: Jepp, tenker iallfall at det burde stå noe om det på den siden der vi forklarer hvordan komme i gang med høsting/skrive beskrivelser

#### Eksempel på bruk av SPARQL

Kopiert fra developers handbook, vet ikke om det er et eksempel som passer bedre

Få oversikt over hvilke egenskaper som finnes:

```sparql
PREFIX dcat: <http://www.w3.org/ns/dcat#>
SELECT *
WHERE {
	?dataset a dcat:Dataset .
    ?dataset ?prop ?value.
}
```

Deretter plukke og velge egenskaper:

```sparql
PREFIX dcat: <http://www.w3.org/ns/dcat#>
PREFIX rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#>
PREFIX dct: <http://purl.org/dc/terms/>
SELECT *
WHERE {​​​​​​​​​​​​​​
 ?dataset a dcat:Dataset .
 ?dataset dct:title ?title.
 ?dataset rdf:type ?type.
# Velg nye egenskaper ved: ?dataset <ønsket egenskap> ?egenskapsverdi
}
```

[Her finner du flere eksempler og informasjon om hvordan du skriver SPARQL-spørringer](https://www.w3.org/TR/rdf-sparql-query/#basicpatterns).

## Finner du ikke det du leter etter?

> Vet du ikke om data du ser etter finnes eller er det data du trenger som ikke er delt ennå? Hvis datasettet ikke er beskrevet på Data.norge.no eller på dataeiers nettsider, bør du gå i dialog med de som tilbyr dem og oppfordre dem til å dele. Du kan bruke Digdirs tjenester til dette, ved å spørre Datajegeren eller etterspørre datasett og API-er i Datalandsbyen.

Reviewer 2: "Datajegeren" og "Datalandsbyen" kan fint være lenker her
```suggestion
Vet du ikke om data du ser etter finnes eller er det data du trenger som ikke er delt ennå? Hvis datasettet ikke er beskrevet på Data.norge.no eller på dataeiers nettsider, bør du gå i dialog med de som tilbyr dem og oppfordre dem til å dele. Du kan bruke Digdirs tjenester til dette, ved å spørre [Datajegeren](https://data.norge.no/forms/nb/data-hunter) eller etterspørre datasett og API-er i [Datalandsbyen](https://datalandsbyen.norge.no).
```

Reviewer: La til lenke direkte til "etterspør datasett og API-er" på datalandsbyen i Nils Ove sitt forslag. 

```suggestion
Vet du ikke om data du ser etter finnes eller er det data du trenger som ikke er delt ennå? Hvis datasettet ikke er beskrevet på Data.norge.no eller på dataeiers nettsider, bør du gå i dialog med de som tilbyr dem og oppfordre dem til å dele. Du kan bruke Digdirs tjenester til dette, ved å spørre [Datajegeren](https://data.norge.no/forms/nb/data-hunter) eller etterspørre datasett og API-er i [Datalandsbyen](https://datalandsbyen.norge.no/category/6/ettersp%C3%B8r-datasett-og-api-er).
```
